templates/person/search.html.twig

```diff
@@ -3,7 +3,7 @@
 {% block pageheader %}
     <h1>Search Persons</h1>
     {% if submitted %}
-       Displaying {{ people|length }} people of {{ people.getTotalItemCount }} total.
```

Comment:
It would be the same information as the table below, so I wonder if it's even necessary? That might be a question for the team to see if there's any reason to have that data at the top and above the table.

Reply:
If it's available elsewhere it's probably easiest to just remove it. If they ask for it to be restored we can do it, but I suspect they won't notice it gone.